Cloud Application Manager/API/centurylink-private-node.md

```diff
@@ -0,0 +1,54 @@
+{{{ "title": "CenturyLink Private Node",
```

Comment:
that kb material was in a source doc with the following title. let's just move it to the appropriate place. <img width="410" alt="screen shot 2017-05-18 at 3 53 47 pm" src="https://cloud.githubusercontent.com/assets/12765541/26223120/5c572598-3be2-11e7-924e-00037fabc45d.png">